src/backend/utilities/emailHandler.ts

```diff
@@ -0,0 +1,55 @@
+import * as path from 'path';
+
+import AWS from 'aws-sdk';
+import nodemailer from 'nodemailer';
+
+import { configuration } from './configuration';
+
+const mailer = nodemailer.createTransport({
+  SES: new AWS.SES(configuration.aws),
+});
+
+const sender = 'Kilt Checkout <checkout@kilt.io>';
```

Comment:
Does it work without quotes? I thought they were part of the syntax. Also the branding spelling is KILT
```suggestion
const sender = '"KILT Checkout" <checkout@kilt.io>';
```

Reply:
It is working without the quotes, but I will correct the spelling

![image](https://user-images.githubusercontent.com/48685760/214553843-1c3b91ba-0e9a-4e4a-8faa-98f974c171bf.png)